resources/dicts/events/ceremonies/ceremony_traits.json

```diff
@@ -9,17 +9,31 @@
         "valor",
         "spirit",
         "intrigue",
-        "observance"
+        "observance",
+        "boldness",
+        "spunk",
+        "valor",
```

Comment:
this is a repeat honor in this category

Reply:
fuck i can't read lmao how did i miss the double valor 😭